aten/src/ATen/native/mps/operations/Copy.mm

```diff
@@ -150,7 +150,11 @@ void copy_cast_mps(at::Tensor& dst, const at::Tensor& src,
         [blitEncoder endEncoding];
 
         if (non_blocking) {
+#if USE_MPSCOMMANDBUFFER
+          stream->commitAndContinue();
```

Comment:
Maybe have the ifdef inside the implementation of commitAndContinue() to avoid duplication? Will reduce changes when we clean up the ifdef.
```
void MPSStream::commitAndContinue() {
   assert(_commandBuffer);
#if USE_MPSCOMMANDBUFFER
   [_commandBuffer commitAndContinue];
#else 
   commit(true);
#endif
 }
```

Reply:
Hmm, I will have to use a different `name` and change it at all call-sites. I think @razarmehr's change refactors bunch of things. So I will let him handle cleanup.